ACKNOWLEDGEMENTS.md

```diff
@@ -0,0 +1,32 @@
+Funding
+=======
+The following institutions, missions, and programs have provided funding
+for pysat development.
+
+Institutions
+------------
+ - The Catholic University of America (CUA)
+ - Cosmic Studio
```

Comment:
@rstoneback, should I update your zenodo institution to Cosmic Studio? Currently Stoneris LLC

Reply:
Please! Thanks for catching that.